RELEASE_PROCESS.rst

```diff
@@ -75,3 +120,13 @@ Merge changes to develop
      git checkout develop
      git merge master
      git push origin develop
+
+Update the release on github
+----------------------------
+
+You can find the release on github at https://github.com/juju/juju-gui/tag/<the newest tag>. Update the
+release notes with the entry from CHANGES.yaml. Upload the dist/jujugui-*.tar.bz2 package as a binary.
+
```

Comment:
Keeping the binary provides a nice archive of what we've done rather than needing to build by checking out the tag and running `make dist` again. I think it would be awesome if we could adapt the upload-release stuff from blues-browser.